pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[project]
+name = "codypy"
+description = "Python wrapper binding for Cody Agent"
+authors = [{ name = "PriNova", email = "info@prinova.de" }]
+readme = "README.md"
+requires-python = ">=3.9"
+keywords = ["cody", "cody-agent", "sourcegraph", "ai", "assistant"]
+dependencies = [
+    "requests",
+    "python-dotenv",
+    "pydantic",
+    "pydantic_core",
+]
+
+[project.optional-dependencies]
+dev = [
+    "black",
+    "isort",
+    "pylint",
+    "ruff",
```

Comment:
Gitlab is trolling me, I have not seen your comment 5 days ago and I explicitly checked. Sorry for that.
Somewhat yes, I personally believe ruff is the future and want to keep running it along with pylint which is much more robust right now (and slow as well). I have only seen a very few marginal things caught by ruff which was not by pylint, and that was likely due to the pylint config we use. If you don't mind, keep it there, otherwise we can stick to pylint.